test/test_backend/test_backend.py

```diff
@@ -211,24 +245,59 @@ def test_cr_hierarchy_data_counts(self):
     #     selected_csets = [cset for cset in related_csets if cset['selected']]
     #     self.assertEqual(selected_csets,[396155663,643758668])
 
-    # TODO: test is failing. fix
-    @unittest.skip("Skipping failing/erroring test temporarily.")
     def test_get_researchers(self):
         """Test get_researchers()"""
```

Comment:
## `test_get_researchers()`
This test was failing primarily because it was expecting some researchers to not be found in our DB, but we've since fetched info for those researchers.